Jenkinsfile

```diff
@@ -135,6 +135,9 @@ pipeline {
                        git clean -xffd
                     """
                     utils.checkout_pr("math", "lib/stan_math", params.math_pr)
+                    if (params.stanc3_bin_url != 'nightly') {
```

Comment:
@serban-nicusor-toptal can you add a textbox in the Build with parameters for `stanc3_bin_url`.